tests/tape/test_jacobian_tape.py

```diff
@@ -606,7 +606,7 @@ def test_non_differentiable_error(self):
             qml.RX(0.543, wires=[0])
             qml.RY(-0.654, wires=[1])
             qml.CNOT(wires=[0, 1])
-            qml.probs(wires=[0, 1])
+            qml.expval(qml.PauliZ(0))
```

Comment:
What was the reason for this change?

Reply:
🤔

Reply:
@albi3ro was this change you or me?

Reply:
I changed it back when we couldn't support calculate the hessian of a vector-valued output.  We can now take the hessian of a vector valued function, so I'm changing it back.

Reply:
@albi3ro in retrospect this was actually a good change, as it increased coverage! I think if you change it back again (sorry 😬 ) codecov will pass and this can be merged in)